sdcm/tester.py

```diff
@@ -233,16 +233,23 @@ def setUp(self):
         self.params['update_db_packages'] = download_dir_from_cloud(update_db_packages)
 
         self.init_resources()
-        self.set_seeds()
 
-        # Init seed nodes
-        self.db_cluster.wait_for_init(node_list=self.db_cluster.seed_nodes)
+        def init_nodes(clus):
+            clus.set_seeds(seeds_selector=self.params.get('seeds_selector'),
```

Comment:
Why do get `seeds_selector`, `seeds_num` as arguments? `params` object is available on cluster objects

Reply:
The function has been removed